lib/src/selector.dart

```diff
@@ -89,6 +98,12 @@ class _Selector0State<T> extends SingleChildState<Selector0<T>> {
     }
     return cache;
   }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<T>('value', value));
```

Comment:
This needs a test

Reply:
You can do something like that: https://github.com/rrousselGit/river_pod/blob/0ba3e28bba005f6be06bf269049a6e762e71998e/packages/flutter_riverpod/test/framework_test.dart#L291